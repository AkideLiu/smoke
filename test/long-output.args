fixtures/bottles.rb
fixtures/bottles
