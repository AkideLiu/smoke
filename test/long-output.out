10
  succeeded
100
  succeeded
1000
  succeeded

3 tests, 0 failures
